tensorboard/webapp/metrics/views/main_view/empty_tag_match_message_component.ts

```diff
@@ -32,13 +32,12 @@ declare namespace Intl {
 
 @Component({
   selector: 'metrics-empty-tag-match-component',
-  template: `No cards matches tag filter
-    <code>/{{ tagFilterRegex }}/</code> among {{ tagCounts | number }} tags<span
-      *ngIf="pluginTypes.size"
-    >
+  template: `No matches for tag filter <code>/{{ tagFilterRegex }}/</code
+    ><span *ngIf="pluginTypes.size">
       and {{ getPluginTypeFilterString(pluginTypes) }} visualization
       filter</span
-    >.`,
+    >
+    out of {{ tagCounts | number }} tags.`,
```

Comment:
FYI, the choice to put tagCount before the visualization time was deliberate since viz filter does not really filter on tags.

Reply:
Yeah, I can see that (mentioned in PR description), but I think as a user the distinction between tags vs cards is not all that helpful, especially since everything else here is referring to filtering "tags", and just having the tag count at the end reads a little better.